Assets/File.php

```diff
@@ -234,92 +234,6 @@ public static function get_shortcodes() {
 	}
 
 	/**
-	 * Replace "[file_link id=n]" shortcode with an anchor tag or link to the file.
-	 *
-	 * @param array $arguments Arguments passed to the parser
-	 * @param string $content Raw shortcode
-	 * @param ShortcodeParser $parser Parser
-	 * @param string $shortcode Name of shortcode used to register this handler
-	 * @param array $extra Extra arguments
-	 * @return string Result of the handled shortcode
-	 */
-	public static function handle_shortcode($arguments, $content, $parser, $shortcode, $extra = array()) {
```

Comment:
Can you please add this to the 4.0.0 changelog under the appropriate section? (Probably under Filesystem API Additions / Changes).